setup/install/index.md

```diff
@@ -14,30 +14,27 @@ in [Quickstart](/setup/quickstart/) might be a better choice.
 
 ## What you'll need
 
-* A machine on which to install Halyard
+* A machine to install Halyard
 
-  This can be a local machine or VM (Ubuntu 14.04/16.04, Debian, or macOS), or
-  it can be a Docker container. Make sure it has at least 12GB of memory.
+  This can be a local machine or a VM (Ubuntu 14.04/16.04, Debian, or macOS), or a Docker container. Whatever you choose, make sure it has at least 12GB of RAM.
```

Comment:
Should this be 4 cores and 16GB of RAM" rather than "12 GB of RAM" to be in line with the requirements for a Kubernetes cluster or the local debiaN?